src/WPConfigTransformer.php

```diff
@@ -65,8 +65,8 @@ public function exists( $type, $name ) {
 		if ( ! trim( $wp_config_src ) ) {
 			throw new Exception( 'Config file is empty.' );
 		}
-
-		$this->wp_config_src = $wp_config_src;
+		// Normalize the newline to prevent an issue coming from OSX
+		$this->wp_config_src = str_replace(array("\n\r", "\r"), array("\n", "\n"), $wp_config_src);
```

Comment:
There's no need for the second array here. Also, we're in PHP 5.4 now and can use short array syntax. Finally, spacing should adhere to WPCS.
```suggestion
		$this->wp_config_src = str_replace( ["\n\r", "\r"], "\n", $wp_config_src );
```